Source/XUnitExtensions/Lit/DiffCommand.cs

```diff
@@ -33,8 +33,8 @@ public class DiffCommand : ILitCommand {
       TextWriter outputWriter, TextWriter errorWriter) {
       var actual = File.ReadAllText(ActualPath);
       if (UpdateExpectFile) {
-        var nonSymlinkedPath = Path.GetFullPath(ExpectedPath).Replace("Source/IntegrationTests/bin/Debug/net6.0/TestFiles/LitTests/LitTest", "Test");
-        File.WriteAllText(nonSymlinkedPath, actual);
+        var path = Path.GetFullPath(ExpectedPath).Replace("bin/Debug/net6.0/", "");
+        File.WriteAllText(path, actual);
```

Comment:
I'm not sure I understand what you're saying. Because of `.Replace("bin/Debug/net6.0/", "")`, it overwrites the `.expect` files in their original location.

Reply:
Ah you're right, I thought it was relying on symlinking inside the output directory. The point stands since some day the "net6.0" part will change, but not worth worrying about now.